tools/ci/check_grep.sh

```diff
@@ -4,95 +4,132 @@ set -euo pipefail
-    echo "ERROR: vareditted cables detected, please remove them."
-    st=1
-fi;
-if grep -Pzo '/obj/structure/machinery/power/apc[/\w]*?\{\n[^}]*?pixel_[xy] = -?[013-9]\d*?[^\d]*?\s*?\},?\n' maps/**/*.dmm ||
-	grep -Pzo '/obj/structure/machinery/power/apc[/\w]*?\{\n[^}]*?pixel_[xy] = -?\d+?[0-46-9][^\d]*?\s*?\},?\n' maps/**/*.dmm ||
-	grep -Pzo '/obj/structure/machinery/power/apc[/\w]*?\{\n[^}]*?pixel_[xy] = -?\d{3,1000}[^\d]*?\s*?\},?\n' maps/**/*.dmm ;	then
 	echo
-    echo "ERROR: found an APC with a manually set pixel_x or pixel_y that is not +-25."
-    st=1
-fi;
-if grep -P '^/area/.+[\{]' maps/**/*.dmm;	then
-    echo "ERROR: Vareditted /area path use detected in maps, please replace with proper paths."
-    st=1
+	echo "${RED}ERROR: incorrect pixel offset variables detected in maps, please remove them.${NC}"
+	st=1
 fi;
+#if grep -P '/obj/structure/cable(/\w+)+\{' maps/**/*.dmm;	then
+#	echo
+#	echo -e "${RED}ERROR: Variable editted cables detected, please remove them.${NC}"
+#	st=1
+#fi;
+#if grep -Pzo '/obj/structure/machinery/power/apc[/\w]*?\{\n[^}]*?pixel_[xy] = -?[013-9]\d*?[^\d]*?\s*?\},?\n' maps/**/*.dmm ||
+#	grep -Pzo '/obj/structure/machinery/power/apc[/\w]*?\{\n[^}]*?pixel_[xy] = -?\d+?[0-46-9][^\d]*?\s*?\},?\n' maps/**/*.dmm ||
+#	grep -Pzo '/obj/structure/machinery/power/apc[/\w]*?\{\n[^}]*?pixel_[xy] = -?\d{3,1000}[^\d]*?\s*?\},?\n' maps/**/*.dmm ;	then
+#	echo "${RED}ERROR: found an APC with a manually set pixel_x or pixel_y that is not +-25.${NC}"
+#	st=1
+#fi;
+#if grep -P '^/area/.+[\{]' maps/**/*.dmm;	then
+#	echo "${RED}ERROR: Vareditted /area path use detected in maps, please replace with proper paths.${NC}"
+#	st=1
+#fi;
 if grep -P '\W\/turf\s*[,\){]' maps/**/*.dmm; then
-    echo "ERROR: base /turf path use detected in maps, please replace with proper paths."
-    st=1
-fi;
-if grep -P '^/*var/' code/**/*.dm; then
-    echo "ERROR: Unmanaged global var use detected in code, please use the helpers."
-    st=1
-fi;
-echo "Checking for whitespace issues"
-if grep -P '(^ {2})|(^ [^ * ])|(^    +)' code/**/*.dm; then
-    echo "space indentation detected"
-    st=1
-fi;
-if grep -P '^\t+ [^ *]' code/**/*.dm; then
-    echo "mixed <tab><space> indentation detected"
-    st=1
+	echo
+	echo "${RED}ERROR: base /turf path use detected in maps, please replace with proper paths.${NC}"
+	st=1
 fi;
-nl='
-'
-nl=$'\n'
-while read f; do
-    t=$(tail -c2 "$f"; printf x); r1="${nl}$"; r2="${nl}${r1}"
-    if [[ ! ${t%x} =~ $r1 ]]; then
-        echo "file $f is missing a trailing newline"
-        st=1
-    fi;
-done < <(find . -type f -name '*.dm')
+#if grep -P '^/*var/' code/**/*.dm; then
+#	echo
+#	echo "${RED}ERROR: Unmanaged global var use detected in code, please use the helpers.${NC}"
+#	st=1
+#fi;
+#echo "Checking for whitespace issues"
+#if grep -P '(^ {2})|(^ [^ * ])|(^	+)' code/**/*.dm; then
+#	echo
+#	echo "${RED}ERROR: space indentation detected.${NC}"
+#	st=1
```

Comment:
honestly we could uncomment this, it should all be gone now

Reply:
handled in #2141